homeassistant/components/keenetic_ndms2/config_flow.py

```diff
@@ -137,8 +136,8 @@ def __init__(self, config_entry: ConfigEntry):
         self._interface_options = {}
 
     async def async_step_init(
-        self, _user_input: dict[str, Any] | None = None
-    ) -> dict[str, Any]:
+        self, _user_input: ConfigType | None = None
```

Comment:
We shouldn't need the underscore prefix for user input variable.

Reply:
It's a common indication of an unused variable. Maybe it's not needed for compile and lint to pass but it's a common good technique for marking such a thing.

BTW this code is unchanged by the PR.

Reply:
This parameter has a standard name `user_input` in our code.

Reply:
Btw, unused parameters are commonly just named `_` and not prefixed with underscore.

Reply:
`_` names are less informative, but you are right for Python prefixing is less popular (dunno why)

At this point, it's easier to make it `user_input` although the IDE blames me... here you go